org.emptyflow.ArdorQuery.yml

```diff
@@ -0,0 +1,26 @@
+app-id: org.emptyflow.ArdorQuery
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+command: ArdorQuery
+modules:
+  - name: anilibria
+    buildsystem: qmake
+    subdir: src
+    build-options:
+      env:
+        PREFIX: /app
+      config-opts:
+        - CONFIG += flatpak
+    sources:
+      - type: git
+        url: https://github.com/trueromanus/ArdorQuery
+        tag: 0.0.11
+        commit: 1b45cb2e5e023eaa7ebc1afd8bf5b35c91bdde4b
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --filesystem=host
```

Comment:
WHy is this needed?

Reply:
Hello, this is because the user can add any file path from the local computer as string in application, and user can also open a file containing the path(s) to other files. Thanks